packages/minter-contracts/ligo/src/fixed_price_sale/fixed_price_sale_market_offchain.mligo

```diff
@@ -0,0 +1,168 @@
+#if !PERMIT_MARKET
+#define PERMIT_MARKET
```

Comment:
consider renaming macro to tell **what** functionality is turned on, not **how** it is implemented.
`PERMIT_MARKET` -> `OFFCHAIN_MARKET`

Reply:
Fixed thanks!